src/request/change-password/index.html

```diff
@@ -0,0 +1,175 @@
+    <script defer bundle-common src="../../lib/CookieJar.js"></script>
+    <script defer bundle-common src="../../lib/BrowserDetection.js"></script>
+    <script defer bundle-common src="../../lib/AccountStore.js"></script>
+    <script defer bundle-common src="../../common.js"></script>
+
+    <script defer bundle-toplevel src="../../components/NoReferrerErrorPage.js"></script>
+    <script defer bundle-toplevel src="../../components/PassphraseBox.js"></script>
+    <script defer bundle-toplevel src="../../components/PassphraseInput.js"></script>
+    <script defer bundle-toplevel src="../../lib/AnimationUtils.js"></script>
+    <script defer bundle-toplevel src="../../lib/I18n.js"></script>
+    <script defer bundle-toplevel src="../../lib/RequestParser.js"></script>
+    <script defer bundle-toplevel src="../../lib/Utf8Tools.js"></script>
+    <script defer bundle-toplevel src="../TopLevelApi.js"></script>
+    <script defer bundle-toplevel src="../../translations/index.js"></script>
+
+    <script defer src="../../lib/Iqons.js"></script>
+    <script defer src="../../lib/LoginFile.js"></script>
+    <script defer src="../../lib/PasswordStrength.js"></script>
+    <script defer src="../../lib/QrEncoder.js"></script>
+    <script defer src="../../components/DownloadLoginFile.js"></script>
+    <script defer src="../../components/LoginFileIcon.js"></script>
+    <script defer src="../../components/ProgressIndicator.js"></script>
+    <script defer src="../../components/PassphraseSetterBox.js"></script>
+    <script defer src="ChangePassword.js"></script>
+    <script defer src="ChangePasswordApi.js"></script>
+    <script defer src="index.js"></script>
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Muli:400,600,700">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY">
+
+    <link rel="stylesheet" bundle-toplevel href="../../../node_modules/@nimiq/style/nimiq-style.min.css">
+    <link rel="stylesheet" bundle-toplevel href="../../../node_modules/@nimiq/style/nimiq-style-icons.min.css">
+    <link rel="stylesheet" bundle-toplevel href="../../nimiq-style.css">
+    <link rel="stylesheet" bundle-toplevel href="../../common.css">
+    <link rel="stylesheet" bundle-toplevel href="../../components/PassphraseInput.css">
+    <link rel="stylesheet" bundle-toplevel href="../../components/PassphraseBox.css">
+
+    <link rel="stylesheet" href="../../components/DownloadLoginFile.css">
+    <link rel="stylesheet" href="../../components/LoginFileIcon.css">
+    <link rel="stylesheet" href="../../components/ProgressIndicator.css">
+    <link rel="stylesheet" href="./ChangePassword.css">
+</head>
+
+<body class="loading">
+    <div class="header-top">
+        <div class="nimiq-app-name">
+            <span class="nimiq-logo"></span>
+        </div>
+    </div>
+
+    <div id="app">
+        <div class="flex-grow"></div>
```

Comment:
The `flex-grow` elements should be unnecessary now with recent `common.css` fixes.